shopfloor_mobile/static/wms/src/components/screen.js

```diff
@@ -176,6 +179,12 @@ Vue.component("Screen", {
                                 <span>{{ $t('app.profile_configure') }}</span>
                             </v-btn>
                         </v-col>
+                        <v-col class="text-center" cols="12">
+                            <v-btn @click="$router.push({'name': 'workstation'})">
+                                <v-icon>mdi-printer</v-icon>
```

Comment:
as the workstation is not just about printers, I'd pick a more generic icon like `cogs` or `account-cog-outline`